features/private-vcs-packages.md

```diff
@@ -37,3 +37,22 @@ Add the Private Packagist repository to your composer.json and require the packa
 </code>
 </pre>
 **Note:** Your composer.lock file contains the download URLs for packages. You need to run composer update at least once to regenerate your composer.lock file before composer install will download files from Private Packagist.
+
+## How do I specify multiple packages in an archive?
```

Comment:
Archive is the wrong word here, would probably use VCS repository instead

Reply:
Or maybe rather something like: `How do I configure a VCS repository with multiple composer.json files?`